src/System.Transactions.Local/src/System/Transactions/TransactionState.cs

```diff
@@ -1533,7 +1516,7 @@ internal override void CreateAbortingClone(InternalTransaction tx)
 
         internal override void GetObjectData(InternalTransaction tx, SerializationInfo serializationInfo, StreamingContext context)
         {
-            throw CreateTransactionAbortedException(tx);
+            throw new PlatformNotSupportedException();
```

Comment:
It is unfortunate that we would need to put this exception back when we start supporting serialization. Maybe comment out the old code and add the PlatformNotSupportedException so we don't need to go back and look for what the code used to be?